lms/static/js/views/fields.js

```diff
@@ -364,7 +364,6 @@
 
             events: {
                 'click': 'startEditing',
-                'change select': 'finishEditing',
```

Comment:
Questions:

Is this something we need to report to Analytics that we are removing?